tests/charm++/zerocopy/Makefile

```diff
@@ -6,9 +6,15 @@ DIRS = \
   dereg_and_nodereg \
```

Comment:
The `largedata` line here should be changed to `large_bcast` to match the directory rename.

Reply:
Done.